pkg/controller/kogitobuild/build/image_stream_kogito.go

```diff
@@ -43,13 +43,13 @@ const (
 )
 
 var (
-	// kogitoImages maps the default Kogito Images on a matrix of RuntimeType and its purpose
-	kogitoImages = map[v1alpha1.RuntimeType]map[bool]string{
+	// KogitoImages maps the default Kogito Images on a matrix of RuntimeType and its purpose
+	KogitoImages = map[v1alpha1.RuntimeType]map[bool]string{
 		v1alpha1.QuarkusRuntimeType: {
 			true:  KogitoQuarkusUbi8s2iImage,
```

Comment:
Just saw that one.... Would it be possible to use `jvm` and `s2i` specific type instead of `true/false` values.

I found it not readable later in the PR: https://github.com/kiegroup/kogito-cloud-operator/pull/479/files#diff-8b7a901c0e8204e44d0c7a73237cd855R117
Did not know what was this true/false about...

Reply:
I think this makes useful to check conditions: 

```
If image[quarkus] then
```

Not sure, have to see the code.